site/docs/v1/tech/apis/two-factor.adoc

```diff
@@ -357,7 +357,7 @@ The unique Id of the user for which to retrieve two-factor status.
 |The user does not have two-factor enabled, or the provided trust is still valid for the next login.
 
 |242
-|The user has two-factor authentication enabled, and will be required to complete a two-factor challenge during the next login attempt.
+|The user has two factor authentication enabled. Since version `1.42.0`, this status code is returned when two factor authentication is required.  The user will be required to complete a two-factor challenge during the next login attempt.
```

Comment:
```suggestion
|The user has two factor authentication enabled. Since version `1.42.0`, this status code is also returned when two factor authentication is required.  The user will be required to complete a two-factor challenge during the next login attempt.
```